src/webpack-plugin/plugin.js

```diff
@@ -0,0 +1,81 @@
+var path = require('path');
```

Comment:
nit: I would rename this file to `index.js` so you can `require('./webpack-plugin'),` in `src/index.js` above, unless there's a strong reason to keep this name due to some webpack convention?

Reply:
There's no reason to keep it like this as far as I know so I'm happy to change it to index.he